### PR TITLE
Tasks 607-610

### PR DESCRIPTION
**Pull request by arlenfan**

### CODE REVIEW

**Comment by swarooprm** [comment]
Reviews: https://github.com/swarooprm/CSE576HW/pull/24

**Comment by swarooprm** [comment]
@arlenfan, could you fix the readme file so that tests pass?

**Comment by arlenfan** [comment]
All checks seem to have passed this time.

File: tasks/task607_sbic_intentional_offense_binary_classification.json [hunk 1]
```diff
@@ -0,0 +1,10430 @@
...
+    "Contributors": [
+        "Arlen Fan"
+    ],
+    "Source": [
+        "SBIC - https://huggingface.co/datasets/social_bias_frames"
+    ],
+    "Categories": [
+        "Binary Classification"
+    ],
+    "Definition": "In this task, you are given a text from a post. Your task is to classify a given post into two categories: 1) yes if the given post is intentionally offensive, 2) no, otherwise. Also, generate label 'no' if offensive statements that appear to be unintentional, or non-offensive statements that are underhandedly offensive. ",
```
Comment (danyaljj) [inline tasks/task607_sbic_intentional_offense_binary_classification.json:11]: Looks good!

**Comment by danyaljj** [comment]
Thanks, the tasks all look good to me!

**Pull request merged**